Tools/CISupport/ews-build/config.json

```diff
@@ -64,6 +64,11 @@
     { "name": "ews133", "platform": "*" },
     { "name": "ews134", "platform": "*" },
     { "name": "ews135", "platform": "*" },
+    { "name": "ews136", "platform": "*" },
+    { "name": "ews137", "platform": "*" },
```

Comment:
you added these two in https://github.com/WebKit/WebKit/pull/7363, so not needed as part of this PR